src/sass/components/_about.scss

```diff
@@ -0,0 +1,163 @@
+.about{
+    background-color: rgba(255, 245, 246, 1);
+    padding-left: 50px;
+    padding-right: 50px;
+ 
+    
+
+
+    &__tittle{
+        font-family: 'Titan One';
+        font-style: normal;
+        font-weight: 400;
+         font-size: 18px;
+         text-align: center;
+        text-transform: uppercase;
+        color: #FFA5BA;
+        padding-top: 125px;
+      
+        
+
+    }
+
+    &__section-tittle{
+        font-family: 'Titan One';
+        font-style: normal;
+        font-weight: 400;
+        font-size: 34px;
+        line-height: 39px;
+         text-align: center;
+         letter-spacing: 0.04em;
+         text-transform: uppercase;
+         color: #D41443;
+        padding-top: 15px;
+      
+    }
+
+         &__position {
+             display: flex;
+             flex-direction: column;
+             justify-content: center;
+             gap: 47px;
+             flex-wrap: nowrap;
+             padding-top: 26px;
+         }
+
+    
+
+
+       
+    
+
+    &__text1{
+        
+        font-style: normal;
+        font-weight: 500;
+        font-size: var(--font-12px);
+        line-height: 162%;
+        color:var(--color-black);
+        display: inline-block;
+        
+    }
+
+    &__text2{
+    font-family: 'DM Sans';
+    font-style: normal;
```

Comment:
DM Sans jest w body, więc nie trzeba go pisać.
font-style: normal to wartość domyślna dla p, więc też nie trzeba tego pisać